packages/nl-design-system-unstable/status-badge-component/README.md

```diff
@@ -0,0 +1,81 @@
+# Status badge
+
+*Status: **In development***
+
+## Introduction
+
+A status badge draws attention and alerts a user to system states. The information in a status badge is short, concise and clearly scannable.
+
+## Variants
+
+- Success
+- Warning
+- Error
+
+## Alternatives and related components
+
+- Status light
+- Text badge
+
+## Anatomy
+
+- Container with padding
+- Standard right and bottom-margin
+- Text
+- Optional icon
+
+## Design tokens
+- --nl-status-badge-border-radius // Border radius
+- --nl-status-badge-text-transform // Uppercase or title case
+- --nl-status-badge-padding-block // Top and bottom padding
+- --nl-status-badge-padding-inline // Left and right padding
+- --nl-status-badge-font-family // Typeface
+- --nl-status-badge-font-weight // Text weight
+- --nl-status-badge-font-size // Text size
+- --nl-status-badge-success-background-color // Background color of success variant
+- --nl-status-badge-success-color // Text color of success variant
```

Comment:
Maybe we can note here in the comment: `// Text color of success variant, should have enough contrast with the --nl-status-badge-success-background-color `

Reply:
Will add this to all three feedback variants!